packages/strapi-plugin-email/admin/src/containers/Settings/index.js

```diff
@@ -0,0 +1,166 @@
+    };
+
+    fetchEmailSettings();
+  }, [formatMessage]);
+
+  return (
+    <>
+      <SettingsPageTitle name={title} />
+      <div>
+        <form>
+          <Header
+            title={{ label: title }}
+            content={formatMessage({ id: getTrad('Settings.subTitle') })}
+            isLoading={showLoader}
+          />
+          <FormBloc
+            title={formatMessage({ id: getTrad('Settings.form.title.config') })}
+            isLoading={showLoader}
+          >
+            <SizedInput
+              disabled
+              label={getTrad('Settings.form.label.defaultFrom')}
+              name="default-from"
+              placeholder={getTrad('Settings.form.placeholder.defaultFrom')}
+              size={{ xs: 6 }}
+              type="email"
+              value={config.settings.defaultFrom}
+            />
+            <SizedInput
+              disabled
+              label={getTrad('Settings.form.label.defaultReplyTo')}
+              name="default-reply-to"
+              placeholder={getTrad('Settings.form.placeholder.defaultReplyTo')}
+              size={{ xs: 6 }}
+              type="email"
+              value={config.settings.defaultReplyTo}
+            />
+            <SizedInput
+              disabled
+              label={getTrad('Settings.form.label.provider')}
+              name="provider"
+              options={providers}
+              size={{ xs: 6 }}
+              type="select"
+              value={config.provider}
+            />
+            <Text>
+              <FormattedMessage
+                id={getTrad('Settings.form.text.configuration')}
+                values={{
+                  file: <code>./config/plugins.js</code>,
+                  link: (
+                    <a
+                      href="https://strapi.io/documentation/v3.x/plugins/email.html#configure-the-plugin"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      link
+                    </a>
+                  ),
+                }}
+              />
+            </Text>
+          </FormBloc>
+          <FormBloc
+            title={formatMessage({ id: getTrad('Settings.form.title.test') })}
+            isLoading={showLoader}
+          >
+            <SizedInput
+              label={getTrad('Settings.form.label.testAddress')}
+              name="test-address"
+              placeholder={getTrad('Settings.form.placeholder.testAddress')}
+              onChange={event => setTestAddress(event.target.value)}
+              size={{ xs: 6 }}
+              type="email"
+              value={testAddress}
+            />
+            <AlignedButton
+              color="success"
+              disabled={testSuccess || !validateEmail(testAddress)}
+              icon={<Envelope style={{ verticalAlign: 'middle' }} />}
+              isLoading={isTestButtonLoading}
+              onClick={handleEmailTest}
```

Comment:
The `onclick` event is not needed here as we will use the `submit` property instead.